api/client/network/create.go

```diff
@@ -55,6 +56,7 @@ func newCreateCommand(dockerCli *client.DockerCli) *cobra.Command {
 	flags.StringSliceVar(&opts.labels, "label", []string{}, "Set metadata on a network")
 	flags.BoolVar(&opts.internal, "internal", false, "Restrict external access to the network")
 	flags.BoolVar(&opts.ipv6, "ipv6", false, "Enable IPv6 networking")
+	flags.BoolVar(&opts.manuallyAttachable, "manually-attachable", false, "Enable manual container attachment")
```

Comment:
No, worker nodes can't create services. Our security model restricts that to only happen from manager nodes.

Also, `docker run --net=container:<some task>` cannot allocate mode resources in the network. Yes it can participate in that network but that is lesser of a problem imo. But if we think that that could be a big enough security risk we can disallow using container networking mode where the parent container is managed.